### PR TITLE
Add configuration annotation to @Enable* annotated classes

### PR DESCRIPTION
**Pull request by hetzijzo**
The main build failed. Forget to add some more @Configuration annotations on some @Enable* classes.

### CODE REVIEW

**Comment by hetzijzo** [comment]
@timtebeek The main branch build failed. I guess not all checks are in place on PR builds. I ran the build, it looks fine now.

File: adding-spring-security/leaveapp-complete/src/main/java/com/jdriven/leaverequest/WebSecurityConfig.java [hunk 1]
```diff
@@ -9,68 +8,68 @@
...
 	}
 
-}
-
-// As per: https://docs.spring.io/spring-security/reference/servlet/oauth2/resource-server/jwt.html#oauth2resourceserver-jwt-authorization-extraction
-class KeycloakRealmRoleConverter implements Converter<Jwt, JwtAuthenticationToken> {
-
-	@Override
+	/**
+	 * As per: <a href="https://docs.spring.io/spring-security/reference/servlet/oauth2/resource-server/jwt.html#oauth2resourceserver-jwt-authorization-extraction">...</a>
```
Comment (timtebeek) [inline adding-spring-security/leaveapp-complete/src/main/java/com/jdriven/leaverequest/WebSecurityConfig.java:46]:
```suggestion
	 * As per: <a href="https://docs.spring.io/spring-security/reference/servlet/oauth2/resource-server/jwt.html#oauth2resourceserver-jwt-authorization-extraction">Extracting Authorities Manually</a>
```

**Comment by timtebeek** [comment]
Thanks again! No idea why the build failure wasn't picked up earlier 🤔

**Comment by timtebeek** [comment]
It appears the workflow does not run for PRs somehow?

**Comment by timtebeek** [comment]
Might be better after https://github.com/timtebeek/spring-security-samples/commit/86e2fc60dfc315ccfe4447587faea81127197109

**Pull request merged**